### PR TITLE
272 local hazard tutorial

### PR DESCRIPTION
**Pull request by ylyangtw**
Please install pyincore branch develop and use the zip file for fully testing.

### CODE REVIEW

**Comment by navarroc** [comment]
For earthquake, I would simplify that to just requesting hazards that match the datasets. We don't support conversion with local hazard so you can't ask for "SD" or "PGV". I would just ask for PGA and 0.2 SA

**Comment by navarroc** [comment]
For tornado, you could add a comment that removing the seed will give random values.

**Comment by ywkim312** [comment]
zip files are under _MACOSX folder. Can we just make a zip file without this _MACOSX folder?

**Comment by navarroc** [comment]
There is no SurgeRaster.tif in the zip file. Also, can we add an example for getting values from the local hurricane? I think you can find an example payload from test_hazard_service.py

**Comment by ylyangtw** [comment]
> _MACOSX

@ywkim312 Seems like it's where mac storing metadata, I couldn't see it on my mac laptop. But I tried compressing it using command `zip -r create_local_hazard.zip create_local_hazard` this time, hopefully this would resolve the issue... Although I couldn't test it myself ^^||

**Comment by ylyangtw** [comment]
@navarroc Thanks much for all the helpful suggestions! I have addressed all of them~

**Comment by ywkim312** [comment]
> > _MACOSX
> 
> @ywkim312 Seems like it's where mac storing metadata, I couldn't see it on my mac laptop. But I tried compressing it using command `zip -r create_local_hazard.zip create_local_hazard` this time, hopefully this would resolve the issue... Although I couldn't test it myself ^^||

Yes it is clean now. I think it is Mac thing. Whenever, there is a zip file from Mac it usually has this __MACOSX thing that is not necessary

**Review by ywkim312 — APPROVED** [review]

**Comment by navarroc** [comment]
Can you add this to the tutorials section in the documentation? See this for an example:
https://incore.ncsa.illinois.edu/doc/incore/tutorials.html

 You can add the notebook itself so it gets rendered and you can add a link to the zip for all the files. See tutorials.md here: https://github.com/IN-CORE/incore-docs/blob/develop/manual_jb/content/tutorials.md

If you aren't sure or don't have time, someone else could pick this up from here.

**Comment by ylyangtw** [comment]
> Can you add this to the tutorials section in the documentation? See this for an example: https://incore.ncsa.illinois.edu/doc/incore/tutorials.html
> 
> You can add the notebook itself so it gets rendered and you can add a link to the zip for all the files. See tutorials.md here: https://github.com/IN-CORE/incore-docs/blob/develop/manual_jb/content/tutorials.md
> 
> If you aren't sure or don't have time, someone else could pick this up from here.

OK, let me try how far I can go

**Comment by ylyangtw** [comment]
@navarroc Just added the notebook/files/zip to the tutorial!

**Comment by navarroc** [comment]
Looks good. I think you also need to make an entry in _toc.yml so it will appear in the side menu

File: manual_jb/content/_toc.yml [hunk 1]
```diff
@@ -57,10 +57,11 @@ chapters:
   sections:
     - file: notebooks/create_dataset/create_dataset.ipynb
     - file: notebooks/create_dfr3_object/create_dfr3_object.ipynb
-    - file: notebooks/create_earthquake/create_earthquake.ipynb
+    - file: notebooks/create_earthquake/create_local_hazard.ipynb
```
Comment (navarroc) [inline manual_jb/content/_toc.yml:60]: Can you change this back? I think you accidentally updated the create_earthquake line as well

Comment (ylyangtw) [reply manual_jb/content/_toc.yml]: Oh yes I noticed the error and changed it back just a few seconds ago :P

**Review by Rashmil-1999 — APPROVED** [review]

**Review by navarroc — APPROVED** [review]

**Review by longshuicy — APPROVED** [review]

**Pull request merged**